x-pack/plugins/observability/public/hooks/slo/use_clone_slo.ts

```diff
@@ -33,34 +32,35 @@ export function useCloneSlo() {
     },
     {
       onMutate: async ({ slo, originalSloId }) => {
-        // Cancel any outgoing refetches (so they don't overwrite our optimistic update)
-        await queryClient.cancelQueries(sloKeys.lists());
+        await queryClient.cancelQueries({ queryKey: sloKeys.lists(), exact: false });
```

Comment:
I'm not sure if the exact option is needed. The documentation states 
> If you don't want to search queries inclusively by query key, you can pass the exact: true option to return only the query with the exact query key you have passed.

But at this point, I'm too scared to remove it again